docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md

```diff
@@ -70,9 +70,9 @@ The following table lists the limits and describes the differences in the suppor
+|Clients use regular expression evaluation that's part of JavaScript. It's provided by the browser and supports a superset of ECMAScript 5.|Clients use the C++ regular expression engine provided as part of the Visual Studio standard template library. This engine complies with ECMAScript 5 standards.|
+|You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex.|Because of the different regex engines, a regex that includes a custom character class based on predefined character classes may return different results in Outlook on Windows (classic) and on Mac versus in Outlook on the web, on mobile devices, and in [new Outlook on Windows](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627).<br/><br/>As an example, the regex `[\s\S]{0,100}` matches any number, between 0 and 100, of single characters that is a whitespace or a non-whitespace. This regex returns different results depending on your Outlook client.<br/><br/>As a workaround, you should rewrite the regex as `(\s\|\S){0,100}`. This regex matches any number, between 0 and 100, of white space or non-white space.<br/><br/>You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex.|
+|An add-in with regular expressions that becomes unavailable in Outlook on Windows (classic) or on Mac (classic) also becomes unavailable on all other supported Outlook clients.|By default, the evaluation of all regular expressions for an add-in is limited to one second. Exceeding this limit causes reevaluation of up to three times. Beyond the reevaluation limit, Outlook on Windows (classic) and on Mac (classic) prevent the add-in from running for the same mailbox in any of the Outlook clients.<br/><br/>Administrators can override these evaluation limits by configuring the `ActivationAlertThreshold` and `ActivationRetryLimit` settings. For guidance on how to configure these settings, see [Evaluation response time for regular expressions in Outlook add-ins](../concepts/resource-limits-and-performance-optimization.md#evaluation-response-time-for-regular-expressions-in-outlook-add-ins).|
```

Comment:
This is only possible if the add-in is disabled, which we no longer do in Win32.